pure_pursuit_controller_node:
  ros__parameters:
    # General controller parameters
    velocity_estimation_topic: /vcu_msgs/velocity_estimation
    steering_topic: /vcu_msgs/steering_feedback
    reference_topic: /planning/reference
    car_command_topic: /control/car_command
    brake_controller_topic: /control/brake_controller
    res_state_topic: /vcu_msgs/res_state
    controller_frequency: 100.0
    controller_timeout_reset: 4000    #Number of steps before the controller resets
    controller_timeout_fatal: 20000   #Number of steps before the controller goes into a fatal error
    braking_acceleration: 20.0
    profiling_enabled: True

    # Pure Pursuit controller parameters
    lookahead_distance_min: 4.0
    lookahead_distance_max: 9.0
    lookahead_distance_factor: 0.4
    l_f: 0.884
    l_r: 0.646
    kp_longitudinal: 2.0
    kd_longitudinal: 0.0
    max_acceleration_output: 5.0
    max_steering: 0.5
    steering_multiplier: 1.0
    velocity_lookahead_distance: 0.0

    mission: "autocross"



inspection_visualization_node:
  ros__parameters:
    mission_finished_topic: /vcu_msgs/mission_finished
